sdk/communication/azure-communication-sms/README.md

```diff
@@ -121,5 +115,9 @@ Check out other client libraries for Azure Communication Services
 [coc]: https://opensource.microsoft.com/codeofconduct/
 [coc_faq]: https://opensource.microsoft.com/codeofconduct/faq/
 [coc_contact]: mailto:opencode@microsoft.com
+[azconfig_docs]: https://docs.microsoft.com/azure/azure-app-configuration
```

Comment:
Here too, for exmple

Reply:
Our docs are at
https://docs.microsoft.com/en-us/azure/communication-services/

Reply:
Thank you, addressed.